src/script_interface/observables/CylindricalProfileObservable.hpp

```diff
@@ -0,0 +1,115 @@
+/*
+  Copyright (C) 2010,2011,2012,2013,2014 The ESPResSo project
+  Copyright (C) 2002,2003,2004,2005,2006,2007,2008,2009,2010
+  Max-Planck-Institute for Polymer Research, Theory Group
+
+  This file is part of ESPResSo.
+
+  ESPResSo is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  ESPResSo is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef SCRIPT_INTERFACE_OBSERVABLES_CYLINDRICALPROFILEOBSERVABLE_HPP
+#define SCRIPT_INTERFACE_OBSERVABLES_CYLINDRICALPROFILEOBSERVABLE_HPP
+
+#include "ScriptInterface.hpp"
+
+#include <memory>
+
+#include "Observable.hpp"
+#include "core/observables/CylindricalProfileObservable.hpp"
+#include "core/observables/CylindricalFluxDensityProfile.hpp"
+
+namespace ScriptInterface {
+namespace Observables {
+
+class CylindricalProfileObservable : public Observable {
+public:
+  const std::string name() const override {
+    return "Observables::CylindricalProfileObservable";
+  };
+
+  VariantMap get_parameters() const override {
```

Comment:
Like how?

Reply:
This would require to refactor `Observable` to use `AutoParameters` (which is not a bad idea in and of itself).  However this would also require to adapt *all* the other observable to use `AutoParameters` which is a daunting task and most probably beyond the scope of this PR.

Reply:
Ah that is true. Than we leave this for know, it is also easily replaced later, I already did several of those.